File: app/src/main/java/com/ruslan/pocketdoc/data/Station.java
package com.ruslan.pocketdoc.data;

import com.google.gson.annotations.SerializedName;

public class Station {

    @SerializedName("Id")
    private String mId;

    @SerializedName("Name")
    private String mName;

    @SerializedName("LineName")
    private String mLineName;

    @SerializedName("LineColor")
    private String mLineColor;

    public String getId() {
        return mId;
    }

    public String getName() {
        return mName;
    }

    public String getLineName() {
        return mLineName;
    }

    public String getLineColor() {
        return mLineColor;
    }
}

File: app/src/main/java/com/ruslan/pocketdoc/data/Station.kt
package com.ruslan.pocketdoc.data

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Station(
        @SerializedName("Id")
        @Expose
        val id: String,

        @SerializedName("Name")
        @Expose
        val name: String,

        @SerializedName("LineName")
        @Expose
        val lineName: String,

        @SerializedName("LineColor")
        @Expose
        val lineColor: String
)